### ISSUE TITLE
Filtering

### ISSUE THREAD

**Issue by multihost** [vatesfr/xo-cli#19]
Hi,

Thanks for adding some filter options yesterday :)

I installed xo-cli on another server now and it does not work as it did yesterday on another server. Have you removed `--list-objects` without `type=`?

```
root@webdev:/home/devslash# xo-cli --list-objects
[
]root@webdev:/home/devslash#
```

**Comment by julien-f** [comment]
Hey, no I did not remove anything and `--list-objects` does not have any requirements on search patterns.

Which version are you using? (just run `xo-cli` without args)

**Comment by julien-f** [comment]
It seems that I have the same issue myself, let me investigate.

**Comment by multihost** [comment]
Yesterday: v0.7.4

Have downloaded the old version from the other server, so its working again now. 

The fault is in: v0.8.1

**Comment by multihost** [comment]
Thanks Julien :D

**Comment by julien-f** [comment]
Fixed in 0.8.2, sorry for that ;)

**Comment by multihost** [comment]
thank you :)
